db/migrate/20210614090147_do_not_materialize_reporting_facilities.rb

```diff
@@ -0,0 +1,30 @@
+class DoNotMaterializeReportingFacilities < ActiveRecord::Migration[5.2]
+  def up
+    drop_view :reporting_patient_states_per_month, materialized: true
+    drop_view :reporting_patient_visits_per_month, materialized: true
+    drop_view :reporting_patient_blood_pressures_per_month, materialized: true
+    drop_view :reporting_facilities, materialized: true
+
+    execute "SET LOCAL TIME ZONE '#{Rails.application.config.country[:time_zone]}'"
+
+    create_view :reporting_facilities, materialized: false
+    create_view :reporting_patient_blood_pressures_per_month, materialized: true
+    create_view :reporting_patient_visits_per_month, materialized: true
+    update_view :reporting_patient_visits_per_month, version: 2, revert_to_version: 1, materialized: true
```

Comment:
This is shoehorned in here, but prevents from having to drop and recreate the patient states table another time.